services/payments/controllers.go

```diff
@@ -158,7 +179,7 @@ func SubmitHandler(service *Service) handlers.AppHandler {
 		// TODO: state machine handling for custodian submissions
 
 		// get the current state of the transaction from qldb
-		resp, err := service.GetTransactionFromDocID(ctx, req.DocumentID)
+		resp, _, err := service.GetTransactionFromDocumentID(ctx, req.documentID)
```

Comment:
The variable names `req` is a bit confusing, the request has been deserialized at this point, would something related to the type `AuthenticatedPaymentState` be clearer?

Reply:
Good call. There are more like this that need to be changed, but I'm going to come back to it once we get all of these branches moved and new tests in. Went ahead and fixed this one.